recipes/tesseract/all/conanfile.py

```diff
@@ -31,50 +47,54 @@ def source(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.options.shared:
+            del self.options.fPIC
```

Comment:
This is usually in configure, not sure if there's a difference

Reply:
Fixed, thanks. options are not evaluated in a `config_options` method